tests/test_networks.py

```diff
@@ -453,3 +453,5 @@ class Model(BaseModel):
 
     assert str(Model(v=NameEmail('foo bar', 'foobaR@example.com')).v) == 'foo bar <foobaR@example.com>'
     assert str(Model(v='foo bar  <foobaR@example.com>').v) == 'foo bar <foobaR@example.com>'
+    assert Model(v=NameEmail('foo bar', 'foobaR@example.com')) == Model(v=NameEmail('foo bar', 'foobaR@example.com'))
+    assert Model(v=NameEmail('foo bar', 'foobaR@example.com')) != 'foo bar <foobaR@example.com>'
```

Comment:
```suggestion
        assert NameEmail('foo bar', 'foobaR@example.com') == NameEmail('foo bar', 'foobaR@example.com')
        assert NameEmail('foo bar', 'foobaR@example.com') != NameEmail('foo bar', 'different@example.com')
```
Your first assertion was more complicated than necessary, the second I'm afraid didn't make any sense since you're comparing a model to a string.

Reply:
Originally wanted a test case to hit that alternate branch, but with the above suggestions not really necessary. Changes applied